build/configuration/pom.xml

```diff
@@ -346,6 +388,9 @@
                <value>upstream</value>
             </property>
          </activation>
+         <properties>
+            <project.build.outputTimestamp>${git.commit.time}</project.build.outputTimestamp>
```

Comment:
it is not working for me. 
```
[INFO] --- artifact:3.5.1:check-buildplan (default-cli) @ infinispan-build-configuration-parent ---
[ERROR] Reproducible Build not activated by project.build.outputTimestamp property: see https://maven.apache.org/guides/mini/guide-reproducible-builds.html
```
I suspect `git.commit.time` is not set when maven checks for it. 

I've replaced with a dummy time and got some progress but, antlr plugin is not compatible with reproducible build.

```
[ERROR] plugin without solution org.antlr:antlr3-maven-plugin:3.5.2, see https://github.com/antlr/antlr3/pull/209
```